docs/tools.md

```diff
@@ -27,6 +27,13 @@ Flags:
 ```
 
 
+## mt-control-server
+
+```
+Usage of ./mt-control-server:
+```
```

Comment:
seems like when `./scripts/dev/tools-to-doc.sh` runs `./mt-control-server -h` it doesn't emit help output. it should, so we can document the config options

Reply:
Hmm, seems like globalconf doesn't add all of the registered flagsets or set the usage. Will circle back on this.

Reply:
I think I'll add a global FlagSet to get an (optional) config file and then include a sample ini file and add usage info to that FlagSet. This is similar to what the `metrictank` cmd does.

Reply:
okay. this area is one where we historically have just done whatever works, and may not be following best practices.
So whatever works.